internal/controllers/image/tests/create-full/00-image.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: create-full
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack-admin
+    secretName: openstack-clouds
+  managementPolicy: managed
+  resource:
+    name: create-full-override
+    # FIXME(mandre) Unable to delete protected image
+    # must call `openstack image set --unprotected <image_name>` prior to deletion
+    protected: false
+    tags:
+      - tag1
+      - tag2
+    visibility: public
+    properties:
+      minDiskGB: 1
+      minMemoryMB: 1
+      hardware:
+        cpuSockets: 1
+        cpuCores: 1
+        cpuThreads: 1
+        cpuPolicy: shared
+        cpuThreadPolicy: prefer
+        cdromBus: scsi
+        diskBus: virtio
+        scsiModel: virtio-scsi
+        vifModel: e1000
+    content:
+      containerFormat: docker
+      diskFormat: qcow2
+      download:
+        url: https://github.com/k-orc/openstack-resource-controller/raw/690b760f49dfb61b173755e91cb51ed42472c7f3/internal/controllers/image/testdata/raw.img.gz
```

Comment:
Nice.

nit: Could we tie this to a branch rather than a commit?

Reply:
I liked that it tied to a commit, because it's predictive and this allows to verify the sha and image size in status.